website/pages/index.tsx

```diff
@@ -89,7 +89,7 @@ const Landing = () => {
                 </LandingCard>
 
                 <LandingCard title="Awesome ecosystem" url="/usage/projects" button="Read more">
-                    In the five years since its release, spaCy has become an industry standard with
+                    In the seven years since its release, spaCy has become an industry standard with
```

Comment:
```suggestion
                    Since its release in 2015, spaCy has become an industry standard with
```
Source: https://explosion.ai/blog/introducing-spacy (maybe we could even link this?)

Reply:
Sounds good to me for now!